### ISSUE TITLE
Scrolling while ctrl is pressed crashing patchmatrix

### ISSUE THREAD

**Issue by Yrds** [OpenMusicKontrollers/patchmatrix#10]
**Steps to reproduce:**

- open patchmatrix normally
- control+scroll

**System Info:**
- kernel: 5.6.11-arch1-1
- mesa 20.0.6-2
- i3wm

**Hardware Info:**
- GPU: Sapphire RX 480 Nitro + (8GBversion)
- CPU: AMD FX 8350

**Logs:**
```
yuri@archlinux ~> LANG=C patchmatrix
patchmatrix 0.18.0
Copyright (c) 2016-2020 Hanspeter Portner (dev@open-music-kontrollers.ch)
Released under Artistic License 2.0 by Open Music Kontrollers
Using visual 0x4DA: R=8 G=8 B=8 A=8 D=24 DOUBLE=0 SAMPLES=0
glewInit failed: Missing GL version
fish: 'patchmatrix' terminated by signal SIGSEGV (Address boundary error)
```
**Comments:**
Opengl works fine here, i can play games and other graphical applications(like blender), so i don't think it's a opengl problem.
Not a big thing, but sometimes bother me.

**Comment by ventosus** [comment]
Thanks for reporting, can reproduce this here.

**Comment by ventosus** [comment]
fixed in f1c7d6b3ff2921514683e823bc380694784668e3
